### PR TITLE
Added yml file generation and verifier class replacing random

### PR DESCRIPTION
**Pull request by rubalgoyal**

### CODE REVIEW

**Comment by charles-moloney** [comment]
Can we delete the files that this is adding that are not part of the business logic changed? Surely we don't need to merge 18,000 file changes to develop

File: benchmarks/Jarcionek/MTG-Deck-Builder/MTG-Deck-Builder-master/src/main/java/mtgdeckbuilder/frontend/CardsDisplayPanel.java [hunk 1]
```diff
@@ -1,43 +0,0 @@
-/** filtered and transformed by PAClab */
-package mtgdeckbuilder.frontend;
```
Comment (jckreg) [inline benchmarks/Jarcionek/MTG-Deck-Builder/MTG-Deck-Builder-master/src/main/java/mtgdeckbuilder/frontend/CardsDisplayPanel.java]: In the last PR we removed the database, suitablePrgrms and benchmardks from the repo. These deletes seem like a duplicate. Is there any way to resolve these locally as they are lagging out the github difff page?

**Comment by rubalgoyal** [comment]
I tried to remove unnecessary files and resolved the conflicts. Please let me know if any other change that needs to be done (I am new to git so learning things).

**Comment by charles-moloney** [comment]
> I tried to remove unnecessary files and resolved the conflicts. Please let me know if any other change that needs to be done (I am new to git so learning things).

No worries! :) It is still saying there are 5000+ files being changed. We have recently configured the .gitignore to ignore files in database, benchmarks, and suitableprgms, but I think this isn't being reflected in your changes because they have already been committed. Our most current version of develop has these files deleted, which is what is creating the conflict. Could you please delete the following directories and commit that change?

/database/
/benchmarks/
/suitablePrgms/

**Comment by rubalgoyal** [comment]
Hi @charles-moloney I took another shot and removed these directories from my last commit using following commands

git rm -rf database/
git rm -rf benchmarks/
git rm -rf suitablePrgms/

Can you check and confirm that these unnecessary files have been removed? Thanks.

File: src/java/transform/Main.java [hunk 2]
```diff
@@ -127,6 +147,7 @@ public static void main(String[] args) throws IOException {
 		}
 		
 		System.out.println(unsuccessfulCompiles + " ------- " + successfulCompiles);
+>>>>>>> ed176acbf9e3813d120e1e688579b6fc99ca5542:src/java/transform/Main.java
```
Comment (charles-moloney) [inline src/java/transform/Main.java]: This seems like a leftover merge conflict marker

**Review by charles-moloney — CHANGES_REQUESTED** [review]
Will advise being careful while merging (especially in transform/Main), but this overall looks good. Thank you for removing the extra files,

File: .project [hunk 3]
```diff
@@ -1,3 +1,22 @@
+<<<<<<< HEAD
```
Comment (jckreg) [inline .project]: There are a couple of other spots that seem to have merge conflict markers left over

File: src/java/transform/visitors/TransformVisitor.java [hunk 4]
```diff
@@ -1133,12 +1156,24 @@ public boolean visit(VariableDeclarationStatement node) {
 		return true;
 	}
 	
+<<<<<<< HEAD:src/java/transform/visitors/TransformVisitor.java
```
Comment (jckreg) [inline src/java/transform/visitors/TransformVisitor.java]: Merge conflict markers, looks like 2 separate instances of HEAD but essentially the same change

**Review by jckreg — CHANGES_REQUESTED** [review]
Charles marked one and I marked 2 other spots with merge conflict markers left over

File: .idea/.gitignore [hunk 5]
```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```
Comment (jckreg) [inline .idea/.gitignore]: I believe the .idea files are from intelliJ or another IDE and shouldn't be included, there is a change in the other PR for adding these to the .gitignore

File: src/java/transform/Main.java [hunk 6]
```diff
@@ -153,7 +175,17 @@ public static void main(String[] args) throws IOException {
 				successfulCompiles.add(file);
 				//unsuccessfulCompiles.remove(file);
 			}
+			
+			if(target.equals("SVCOMP")) {
+				prepareForSvcompBenchmark(file);
+			}
 		});
+		//TODO: Temporary added some extra Print statements
```
Comment (jckreg) [inline src/java/transform/Main.java]: Are these still temporary print statements? I like having them in personally on the runs that I have done. If they are temporary should they be removed or should we just remove the TODO and keep the print statements? Any thoughts?

File: svcomp-benchmarks/.DS_Store
Comment (jckreg) [inline svcomp-benchmarks/.DS_Store:1]: We had recently added .DS_Store files to the .gitignore. Are these files needed?

File: src/java/transform/Main.java (hunk 6 above)
Comment (rubalgoyal) [reply src/java/transform/Main.java]: I removed the //TODO comment.

**Comment by rubalgoyal** [comment]
Hi @charles-moloney  and @jckreg thanks for the review and feedback. I have incorporated the changes suggested. Please let me know if anything is remaining.

Further, I have included my manually converted benchmarks for the SVCOMP submission in the folder **svcomp-benchmarks**.

File: svcomp-benchmarks/.DS_Store
Comment (jckreg) [inline svcomp-benchmarks/.DS_Store:1]: Please remove this

File: svcomp-benchmarks/ArrayBasedCollectionTimer/.DS_Store
Comment (jckreg) [inline svcomp-benchmarks/ArrayBasedCollectionTimer/.DS_Store:1]: Please Remove this

File: svcomp-benchmarks/Base64/.DS_Store
Comment (jckreg) [inline svcomp-benchmarks/Base64/.DS_Store:1]: please remove this

File: svcomp-benchmarks/.idea/codeStyles/Project.xml
Comment (jckreg) [inline svcomp-benchmarks/.idea/codeStyles/Project.xml:1]: If we could get rid of the .idea folder and its contents as well that would be great

**Review by jckreg — COMMENTED** [review]
These files should be removed

**Review by jckreg — DISMISSED** [review]

**Review by charles-moloney — APPROVED** [review]

**Review by jckreg — APPROVED** [review]

**Pull request merged**